pkg/globalconfig/global_config.go

```diff
@@ -239,6 +248,12 @@ func ReadGlobalConfig() error {
 		return err
 	}
 
+	// Get the legacy project list if there is one
+	err = yaml.Unmarshal(source, &ddevLegacyGlobalConfig)
+	if err != nil {
+		return err
+	}
```

Comment:
We do this in `ReadGlobalConfig` because this is the best point to guarantee that:
1. We haven't overwritten the config file yet
2. We haven't run `ReadProjectList` yet